build/build-allocation-images/go/gen.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+export GO111MODULE=on
+
+mkdir -p /go/src/
+cp -r /go/src/agones.dev/agones/vendor/* /go/src/
+
+cd /go/src/agones.dev/agones
+go install -mod=vendor github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
+go install -mod=vendor github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
+
+outputpath=./allocation/go
```

Comment:
I think this should go under `pkg/allocation/go`. The generated code for `sdk.proto` doesn't live at the top level of the repo; it lives under the pkg directory.